src/controller/data_model/controller-clusters.matter

```diff
@@ -4018,31 +4012,46 @@ provisional cluster BooleanSensorConfiguration = 128 {
     kSensitivityLevel = 0x8;
   }
 
+  bitmap SensorFaultBitmap : bitmap16 {
+    kGneralFault = 0x1;
```

Comment:
I guess this is generated, so if we fix the XML this is also fixed?

Reply:
Yes, will be updated.